platform/platform_linux.cpp

```diff
@@ -25,17 +25,13 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 
-#include <arpa/inet.h>
+#include <netdb.h>
 #include <linux/limits.h>  // PATH_MAX
-#include <netinet/in.h>
 
 #include <QStandardPaths>  // writableLocation GenericConfigLocation
 
 namespace
 {
-// Web service ip to check internet connection. Now it's a GitHub.com IP.
-char constexpr kSomeWorkingWebServer[] = "140.82.121.4";
```

Comment:
Please use cdn.organicmaps.app 's ip address instead to simplify the code and for faster checks.

Reply:
Does it mean that `65.108.198.117` will be a long term available static IP?

Reply:
We'll update it if it changes for whatever reason.

Reply:
Done for now. I put the definition into the usual header, so it is not burried in the codebase, and close to where `cdn.organicmaps.app` is also set.